internal/saucecloud/cloud.go

```diff
@@ -102,39 +107,58 @@ func (r *CloudRunner) collectResults(results chan result, expected int) bool {
 	return passed
 }
 
-func (r *CloudRunner) runJob(opts job.StartOptions) (job.Job, error) {
+func (r *CloudRunner) runJob(opts job.StartOptions) (job.Job, bool, error) {
```

Comment:
Would be good to add a method description now. With the third parameter added there, it's unclear what the `bool` parameter indicates.